third_party/skia/platform_view_rtree_unittests.cc

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright 2012 Google Inc.
+ *
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+#include "flutter/testing/testing.h"
+#include "flutter/testing/thread_test.h"
+#include "skia/platform_view_rtree.h"
+
+#include "third_party/skia/include/core/SkCanvas.h"
+#include "third_party/skia/include/core/SkPictureRecorder.h"
+
+namespace flutter {
+namespace testing {
+
+TEST_F(PlatformViewRTree, NoIntersection) {
```

Comment:
I don't see the GN rule that compiles this TU. Did you forget to check that in? This would require its own test harness as it is now no longer in the `//flutter/lib/ui`. So please don't forget to add that to the `run_tests.py`.